### DISCUSSION TITLE
Seguridad

### DISCUSSION THREAD

**Discussion by Juan3728825** [community/community#68359] [Code Security] [answered]
### Select Topic Area

Question

### Body

Tengo un problema y es qué no sé con qué puedo trabajar para las notificaciones de una pagina web que estoy haciendo, he intentado de todo y nada ha funcionado

**Comment by Juan3728825** [comment, marked as answer]
Respondiendo a mi pregunta, he conseguido trabajar con webhook y ha salido efectivo, gracias por la atención

**Comment by Juan3728825** [comment]
también y adicional he encontrado la manera de consumir la API de mercado libre mediante la busqueda de los webhook para mis notificaciones

**Comment by miguelvele** [comment]
Parece ser qué se ha respondido a si mismo su pregunta pero la incógnita es, cómo logró acceder a la webhook?
